pkg/cmd/runtime/main.go

```diff
@@ -33,14 +33,16 @@ func Main(cfg *config.Config) {
 
 	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", cfg.Runtime.Port))
 	if err != nil {
-		log.Fatalf("failed to listen: %v", err)
+		err = errors.WithStack(err)
+		logger.Fatalf("failed to listen: %v", err)
```

Comment:
What is the different between %+v and %v?

Reply:
`%+v` is the extended format. Each Frame of the error's StackTrace will be printed in detail.

https://godoc.org/github.com/pkg/errors

Reply:
got it, the code here should to be +v or not?

Reply:
fixed